### PR TITLE
ingress metrics debugging

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: test/extended/prometheus/prometheus.go [hunk 1]
```diff
@@ -250,6 +250,18 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			defer func() {
+				dump := func(resources ...string) {
+					for _, resource := range resources {
+						out, err := oc.AsAdmin().Run("get").Args("--namespace", "openshift-ingress", resource, "-o", "json").Output()
+						if err == nil {
+							e2e.Logf(out)
```
Comment (Miciah) [inline test/extended/prometheus/prometheus.go]: Use `e2e.Logf("%s", out)`.

Comment (ironcladlou) [reply test/extended/prometheus/prometheus.go]: Fixed

**Pull request merged**